build-system/tasks/e2e/expect.js

```diff
@@ -127,7 +123,7 @@ const chaiMethodsAndProperties = [
   {name: 'oneOf', type: ChaiType.METHOD},
   {name: 'ownProperty', type: ChaiType.METHOD},
   {name: 'ownPropertyDescriptor', type: ChaiType.METHOD},
-  {name: 'property', type: ChaiType.METHOD},
+  {name: 'property', type: ChaiType.CHAINABLE_METHOD},
```

Comment:
Why should this be chainable method? It's not implemented that way in Chai

https://github.com/chaijs/chai/blob/42509fac016182ddde8b4edf9ff3ff1ce285cebf/lib/chai/core/assertions.js#L1885

```js
Assertion.addMethod('property', assertProperty);
```

Reply:
I'm not sure, actually. I narrowed down the cause of the error to this line and found that changing the method fixed it.

Reply:
huh weird. I think a comment could help if there's weirdness with this later, but if you're ready to merge now don't let that stop you

Reply:
Good idea, added a comment! Thanks.